src/services/tokenPriceService.ts

```
@@ -0,0 +1,106 @@
+export class TokenPriceService {
+  private static readonly GECKO_TERMINAL_BASE_URL =
+    'https://api.geckoterminal.com/api/v2/networks/polygon_pos/tokens';
+
+  static async fetchTokenPrice(tokenAddress: string): Promise<{
+    price: number;
+    priceUSD: number;
+    marketCap: number | null;
+  } | null> {
+    try {
+      const response = await axios.get<GeckoTerminalResponse>(
+        `${this.GECKO_TERMINAL_BASE_URL}/${tokenAddress.toLowerCase()}/`,
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)